docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+  redis:
+    image: redis
+    volumes:
+      - redis-data:/data
+  web:
+    build: .
+    command: bundle exec rails s
+    depends_on:
+      - db
+      - redis
+    environment:
+      - RAILS_ENV=production
+      - DATABASE_URL=postgresql://postgres:@db:5432/intercity
+      - SECRET_KEY_BASE=a
+      - RAILS_SERVE_STATIC_FILES=true
+      - REDIS_URL=redis://redis:6379
+      - HOSTNAME=intercity.example.com
```

Comment:
Are these values overridable?

Reply:
@jvanbaarsen Yep! You can specify them when executing the `docker-compose` command. Currently this file would only be suitable for development so some sane example defaults in there would be good.

Reply:
👍